_posts/2020/11/2020-11-06-configure-artipie-maven-repo.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+date: 2020-11-06
+title: "Using Artipie as a maven repository"
+author: Alexander Krasnov
+description: This blog post explains how to use Artipie as a Maven repository.
+keywords:
+  - artipie
+  - private
+  - maven
+  - repository
+  - registry
+  - java
+---
+
+## Introduction
+
+Nowadays Apache Maven is [one of the most popular](https://www.baeldung.com/java-in-2019#build-tools-adoption) build management tool for Java. Many companies that do Java development have private Maven repositories, where they store all the artifacts. For example, the team does not work in open source and has some private artifacts. This team needs to share easily and quickly among themselves and with other developers from their company. Or you might be wondering how to set up an own repository for a pet project. [Artipie](https://github.com/artipie/artipie) allows you to solve this problem as it is a binary artifact management tool. In the blog post I will share with you how to configure Artipie and project for this purpose. Let's start!
```

Comment:
@genryxy the start of the blog post feels like an introduction of some scientific article. I think it should be more lively, addressing an actual person who may need Artipie in team/company. Your reader already did choose Java and is using Maven as build tool, you do not need to explain why Java/Maven are popular. Rather I'd suggest you to state problem our client may solve using Artipie: breaking some big repository into multiple repositories possibly developed by different people/teams. That looks like case when team/company going to need and artifact manager. Other one would be creating local cache for build artifacts from Maven Central, it might be the theme for another blog post.